assets/css/styles.css

```diff
@@ -217,9 +219,9 @@ a.dropdown-item:active {
 02. HEADER
 *===================================*/
 header {
-  padding: 25px 65px;
+  padding: 15px 65px;
```

Comment:
Don't use absolute sizing. Please change this to relative. 
https://www.w3schools.com/CSSref/css_units.asp